### PR TITLE
TCT-ColBERTv2 update on MS MARCO V2 - trained models

### PR DESCRIPTION
**Pull request by lintool**

### CODE REVIEW

File: docs/experiments-msmarco-v2-tct_colbert-v2.md [hunk 1]
```diff
@@ -8,20 +8,32 @@ The model is described in the following paper:
...
 ```
 
-## Passage V2
+These are indexes and encoder for the retrained (V2) models:
 
-Dense retrieval with TCT-ColBERT-V2, brute-force index:
+```bash
+export PASSAGE_INDEX1="/store/scratch/indexes/trec2021/faiss-flat.tct_colbert-v2-hnp.psg_v2_ft.msmarco-passage-v2-augmented"
+export DOC_INDEX1="/store/scratch/indexes/trec2021/faiss-flat.tct_colbert-v2-hnp.psg_v2_ft.msmarco-doc-v2-segmented"
+export ENCODER1="/store/scratch/indexes/trec2021/torch_ckpt/tct_colbert-v2-hnp-msmarco-hn-msmarcov2"
```
Comment (justram) [inline docs/experiments-msmarco-v2-tct_colbert-v2.md:31]: We've uploaded the checkpoint to hgf: [modelcard](https://huggingface.co/castorini/tct_colbert-v2-hnp-msmarco-r2).
Maybe we can revise ENCODER1 as `castorini/tct_colbert-v2-hnp-msmarco-r2`?

**Review by justram — APPROVED** [review]
LG and ready to merge.
There's only one minor suggestion.

File: docs/experiments-msmarco-v2-tct_colbert-v2.md (hunk 1 above)
Comment (lintool) [reply docs/experiments-msmarco-v2-tct_colbert-v2.md:31]: Yup, will do!

File: docs/experiments-msmarco-v2-tct_colbert-v2.md [hunk 2]
```diff
@@ -53,7 +65,35 @@ However, we measure recall at both 100 and 1000 hits; the latter is a common set
 Because there are duplicate passages in MS MARCO V2 collections, score differences might be observed due to tie-breaking effects.
 For example, if we output in MS MARCO format `--output-format msmarco` and then convert to TREC format with `pyserini.eval.convert_msmarco_run_to_trec_run`, the scores will be different.
 
-## Document V2
+## Passage V2 (Retrained)
+
+Dense retrieval with TCT-ColBERTv2 model retrained on MS MARCO (V2) passage data, with FAISS brute-force index:
```
[the remark this answers is not in the document]
Comment (lintool) [reply docs/experiments-msmarco-v2-tct_colbert-v2.md:70]: okay, will do!

**Comment by lintool** [comment]
Doing final check, then I'll merge.

**Review by jacklin64 — APPROVED** [review]

**Pull request merged**